cmd/sdkch/README.md

```diff
@@ -0,0 +1,174 @@
+# sdkch
+Simple tool to maintain modular changelogs
+
+# Usage
+
+```
+$ sdkch -help
+usage: sdkch COMMAND
+
+Maintain unreleased changelog entries in a modular fashion.
+
+Commands:
+    init
+    add SECTION STANZA            Add an entry file.
+                                  Read from stdin until it
+                                  encounters EOF.
+    generate [VERSION]            Generate a changelog in
+                                  Markdown format and print it
+                                  to stdout. VERSION defaults
+                                  to UNRELEASED.
+
+    Sections             Stanzas
+         ---                 ---
+    breaking                gaia
+    features             gaiacli
+improvements            gaiarest
+    bugfixes                 sdk
+                      tendermint
+
+Flags:
+  -d string
+    	entry files directory (default "./.pending")
+```
+
+## Initialise a new entry files directory
+
+First, you need to initialise a new directory that will host entry files:
+
+```bash
+$ sdkch init
+```
+
+A new hidden directory `.pending` is created. Its 2-level structure looks like the following:
+
+```
+$ tree .pending/
+.pending/
+|-- breaking
+|   |-- gaia
+|   |-- gaiacli
+|   |-- gaiarest
+|   |-- sdk
+|   `-- tendermint
+|-- bugfixes
+|   |-- gaia
+|   |-- gaiacli
+|   |-- gaiarest
+|   |-- sdk
+|   `-- tendermint
+|-- features
+|   |-- gaia
+|   |-- gaiacli
+|   |-- gaiarest
+|   |-- sdk
+|   `-- tendermint
+`-- improvements
+    |-- gaia
+    |-- gaiacli
+    |-- gaiarest
+    |-- sdk
+    `-- tendermint
+
+24 directories, 0 files
+```
+
+Deepest level directories can contain `entry files`. Entry files are created before submitting a PR for review.
+
+## Add a new entry
+
+You can either drop a text file in the appropriate directory or use the `add` command:
+
+```bash
+$ sdkch add features sdk
+[\#3813] New sdk.NewCoins safe constructor to replace bare
```

Comment:
Does this open a paged reader like `git commit`? That'd be cool if it did. `-m` could take the entry message directly.

Reply:
That's a good idea. One could either use `-m` - and be careful about escaping characters correctly - or let the command open the editor specified in the `VISUAL`/`EDITOR` global environment variables [1]

[1] http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html

Reply:
Actually, I'd do it in a separate PR if we think it's really necessary. The command creates a file which you can always edit/rename later on. Is that acceptable?

Reply:
Yes, can we do this? Many times I like to have bullet points and such and I can see that being tedious coming through just STDIN.

Reply:
I **do** think it's necessary for a good UX flow, but it can be a separate PR sure. Do you know how to do this? I can submit a PR.

Reply:
Yes, I do. I see your point re: bullet points, will address asap.

Reply:
I think we should probably address as a part of this PR

Reply:
I'm on it - will update developers reference and mention of sdkch

Reply:
Done, please test - especially with vim as it might require a proper TTY upon execution.